gtsam/discrete/DecisionTree.h

```diff
@@ -340,4 +340,11 @@ namespace gtsam {
     return f.apply(g, op);
   }
 
+  /// unzip a DecisionTree if its leaves are `std::pair`
+  template<typename L, typename T1, typename T2>
+  std::pair<DecisionTree<L, T1>, DecisionTree<L, T2> > unzip(const DecisionTree<L, std::pair<T1, T2> > &input) {
```

Comment:
Actually, @ProfFan , this should probably have been a static method and be called `Unzip`. Or a method, and then it would not take an argument, right?

Reply:
BTW, just looking for consensus, I think I would change it to static.

Reply:
Actually, I see now it is neither: it is a free function :-/ Which is correctly named, so never mind! Will re-format is all.

Reply:
Do you want to reopen CI?

Reply:
Yes, I'll just do the required one and merge